packages/flutter_driver/test/src/real_tests/test_organisation_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path/path.dart' as p;
+
+void main() {
+  // On CI we only execute the tests in the `test/src/real_test` and
```

Comment:
Should we just move this to dev/bots/test.dart?

Reply:
I think this is a better location for the test because I will see it fail during local development when messing up the organisation and doing a `flutter test`. Moving it to the bots script means I'll only learn about it during presubmit. I prefer to be notified as early as possible (and before sending something out for review).